server_admin/topics/groups/default-groups.adoc

```diff
@@ -1,12 +1,12 @@
 
 === Default Groups
 
-Default groups allow you to automatically assign group membership whenever any new user is created or imported through
-<<_identity_broker, Identity Brokering>>.
-To specify default groups go to the `Groups` left menu item, and click the `Default Groups` tab.
-
-.Default Groups
-image:{project_images}/default-groups.png[]
+Default groups allow you to automatically assign group membership to any new user created or imported through <<_identity_broker, Identity Brokering>>.
```

Comment:
Allow you is product focused, such as "the accelerator on your new Porsche allows you to go from 0 to 60 in ten seconds" instead of "to go from 0 to 60..,"
How about:  "To automatically assign group membership to new users created or imported through <<_identity_broker, Identity Brokering>>, you use default group."